.drone.yml

```diff
@@ -61,6 +61,19 @@ pipeline:
       branch: master
       event: push
 
+    trigger_deploy_to_dev:
+      image: quay.io/ukhomeofficedigital/drone-trigger:v0.3.0
+      drone_server: https://drone-gitlab.acp.homeoffice.gov.uk
+      repo: modern-slavery/kube-modern-slavery
+      secrets:
+        - drone_token
```

Comment:
Added the secret on dev environment. Hope this is what you meant.

Reply:
The way you refer to it here is fine but the secret itself doesn't appear to be in Drone.

Reply:
Thanks, I have added it to the drone repo.